recipe/meta.yaml

```diff
@@ -12,10 +12,14 @@ source:
     sha256: {{ sha256 }}
 
 build:
-    number: 5
+    number: 6
+{% if build_type == 'cuda' %}
     string: cuda{{ cudatoolkit | replace(".*", "") }}_{{ PKG_BUILDNUM }}
     script_env:
       - CUDA_HOME
+{% else %}
+    string: {{ PKG_BUILDNUM }}
```

Comment:
I think conda will by default use hash and build number in the string for this case. So probably we don't need this.

Reply:
Yes, done.